### PR TITLE
Add Vazir font for Persian glyphs

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

**Comment by john-preston** [comment]
Thanks!

File: ui/style/style_core_font.cpp [hunk 1]
```diff
@@ -300,11 +314,9 @@ FontData::FontData(int size, uint32 flags, int family, Font *other)
 	f.setUnderline(_flags & FontUnderline);
 	f.setStrikeOut(_flags & FontStrikeOut);
 
-#ifdef DESKTOP_APP_USE_PACKAGED_FONTS
 	if (_flags & FontSemibold) {
-		f.setWeight(QFont::DemiBold);
+		f.setStyleName("Semibold");
```
Comment (hexchain) [inline ui/style/style_core_font.cpp:318]: May I ask why this is required? It breaks styles for fallback fonts on Linux again. Looks like Qt does not translate `styleName` to style flags, so fonts without a "Semibold" style will not be used regardless.

Comment (ilya-fedin) [reply ui/style/style_core_font.cpp:318]: They shouldn't be used

Comment (ilya-fedin) [reply ui/style/style_core_font.cpp:318]: Now bold-styled messages should be bold, but semibold UI elements that rendered throught font which doesn't have semibold should be regular, on all platforms

Comment (ilya-fedin) [reply ui/style/style_core_font.cpp:318]: Also, open sans should be always when tdesktop is installed, custom fonts aren't supported

Comment (ilya-fedin) [reply ui/style/style_core_font.cpp:318]: Anyway, a new release should be soon with semibold style name as a priority and bold style as a fallback
https://github.com/desktop-app/lib_ui/tree/bold

**Pull request merged**